helm/cups/templates/deployment.yaml

```diff
@@ -37,6 +37,15 @@ spec:
               subPath: cupsd.conf
             - name: empty
               mountPath: /etc/cups/
+        - name: printers-conf
+          image: busybox
```

Comment:
I think this can be combined into a single init container and adding additional commands. Can you test that @zengwuxin?